web/libs/editor/src/components/HtxTextBox/HtxTextBox.js

```diff
@@ -85,9 +85,13 @@ export class HtxTextBox extends React.Component {
   };
 
   updateHeight = throttle(() => {
-    const height = this.inputRef.current?.scrollHeight || 0;
+    // very important to add borders to the height, otherwise input will be shrinking on every recalc
+    const borders = 2;
```

Comment:
```suggestion
    const borders = 2;
```
We should compute the border contributed height from the element once, otherwise if there ever were to be a change that was made via css or inline style that effected the border width of the element, this would regress back to the original bug.

Reply:
now I set border-width in the code to always have it consistent